### PR TITLE
Strategic framework page

### PR DESCRIPTION
**Pull request by davidrapson**
Main changes here:

- (Re)adds basic support for content segments in templates
- Reworks case-study card component
- Allows case studies to be shown on information pages
- Updates type styles ~(need to check these with Chloe first)~

~This PR includes some placeholder content whilst until I've modelled it in the CMS.~

image

### CODE REVIEW

File: services/content-api.js [hunk 1]
```diff
@@ -83,7 +83,23 @@ function getListingPage({ locale, path }) {
         },
         json: true
     }).then(response => {
-        return response.data.map(item => item.attributes).find(_ => _.path === path);
+        const attributes = response.data.map(item => item.attributes);
+        const match = attributes.find(_ => _.path === path);
+
+        if (match) {
+            const exampleCaseStudy = {
```
Comment (davidrapson) [inline services/content-api.js]: Placeholder data, will be removed.

File: middleware/securityHeaders.js [hunk 2]
```diff
@@ -63,7 +63,8 @@ function defaultSecurityHeaders() {
         '*.twimg.com',
         '*.youtube.com',
         'sentry.io',
-        '*.biglotteryfund.org.uk'
+        '*.biglotteryfund.org.uk',
+        'via.placeholder.com'
```
Comment (davidrapson) [inline middleware/securityHeaders.js]: Placeholder data, will be removed.

File: views/pages/toplevel/over10k.njk [hunk 3]
```diff
@@ -17,40 +17,42 @@
...
             titleText = copy.title,
             image = heroImage,
-            accent = 'turquoise'
+            accent = pageAccent
         )
     }}
 
     <article role="main" class="nudge-up">
-        <div class="content-box inner--wide-only accent--turquoise a--border-top">
+        <div class="content-box inner--wide-only accent--{{ pageAccent }} a--border-top">
```
Comment (mattandrews) [inline views/pages/toplevel/over10k.njk]: 💯

File: controllers/routes.js [hunk 4]
```diff
@@ -227,9 +227,12 @@ sections.about.addRoutes({
             '/england/about-big',
             '/wales/about-big',
             '/scotland/about-big',
-            '/northernireland/about-big',
+            '/northernireland/about-big'
         ]
     }),
+    content: cmsRoute({
+        path: '/*'
```
Comment (mattandrews) [inline controllers/routes.js]: This seems a bit too general, but does this just mean that any page under `/about/` will first be tried against the CMS, then against the legacy site, then as a 404?

Comment (davidrapson) [reply controllers/routes.js]: Yep, exactly.

Comment (mattandrews) [reply controllers/routes.js]: Carry on!

**Comment by davidrapson** [comment]
@mattandrews Mind giving this a review again, think this is ready to go now.

**Pull request merged**